src/editors/containers/VideoEditor/components/VideoSettingsModal/components/VideoSourceWidget/index.jsx

```diff
@@ -52,7 +52,12 @@ export const VideoSourceWidget = ({
       [widgetHooks.selectorKeys.allowVideoSharing]: widgetHooks.genericWidget,
     },
   });
-  const { updateVideoId, updateVideoURL } = hooks.sourceHooks({ dispatch });
+  const { videoIdChangeAlert } = hooks.videoIdChangeAlert();
+  const { updateVideoId, updateVideoURL } = hooks.sourceHooks({
+    dispatch,
+    previousVideoId: videoId.formValue,
+    videoIdChangeAlert,
```

Comment:
nit: what do you think about something like this to better show what is being used by the hook?
```setAlert: videoIdChangeAlert.set```